src/__tests__/components/App.test.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
+// import ReactDOM from 'react-dom';
```

Comment:
need that line there?

Reply:
will keep it till the old version of test case is gone